patch/0001-SPARK-LLAP-RANGER-Integration_2.2.0.patch

```diff
@@ -0,0 +1,382 @@
+From d8e5e81c14fbbe4a515aeff1352df766b1bb0c17 Mon Sep 17 00:00:00 2001
+From: Mingjie Tang <mtang@hortonworks.com>
+Date: Tue, 2 May 2017 15:58:56 -0700
+Subject: [PATCH] add the llap for the ranger
+
+fix the compile scala style type issue
```

Comment:
Please remove this redundant descrptions in line 6 ~ line 22. Please refer the other patches.